README.md

```diff
@@ -0,0 +1,80 @@
+sbt-sqs
+==============
+Support for running [ElasitcMQ](https://github.com/adamw/elasticmq) in tests.
+
+Installation
+------------
+Add the following to your `project/plugins.sbt` file:
+```
+addSbtPlugin("com.localytics" % "sbt-sqs" % "0.1.0")
+```
+
+sbt 0.13.6+ is supported. 0.13.5 should work with the right bintray resolvers.
+
+Usage
+-----
+To use ElasticMQ in your project you can call `start-elastic-mq` and `stop-elastic-mq` directly in `sbt`.
+
+To have ElasticMQ automatically start and stop around your tests
+```
+startElasticMQ <<= startElasticMQ.dependsOn(compile in Test)
+test in Test <<= (test in Test).dependsOn(startElasticMQ)
+(test in Test) <<= ((test in Test), stopElasticMQ) { (test, stop) => test doFinally stop }
+```
+
+`startElasticMQ` will download an ElasticMQ jar if one is not already present in the `elasticMQDir`.
+
+Configuration
+-------------
+**Setup**
+
+Set the download directory for the jar. Defaults to "elastic-mq".
+```
+elasticMQDir := file("dir-name")
+```
+
+Set the version of ElasticMQ to download. Defaults to "0.9.0-beta1"
+```
+elasticMQVersion := "0.8.12"
```

Comment:
There is a note in the queues section about this but it's probably a good idea to add something here as well.